tools/FakeItEasy.Build/Program.cs

```diff
@@ -0,0 +1,157 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.CompilerServices;
+using static Bullseye.Targets;
+using static SimpleExec.Command;
+
+namespace FakeItEasy.Build
+{
+    class Program
+    {
+        private const string TestsDirectory = "./artifacts/tests";
+        private const string Solution = "./FakeItEasy.sln";
+        private const string VersionInfoFile = "./src/VersionInfo.cs";
+        private const string RepoUrl = "https://github.com/FakeItEasy/FakeItEasy";
+
+        private static readonly string[] ProjectsToPack =
+        {
+            "./src/FakeItEasy/FakeItEasy.csproj",
+            "./src/FakeItEasy.Analyzer/FakeItEasy.Analyzer.CSharp.csproj",
+            "./src/FakeItEasy.Analyzer/FakeItEasy.Analyzer.VisualBasic.csproj"
+        };
+
+        private const string AnalyzerMetaPackageNuspecPath = "./src/FakeItEasy.Analyzer.nuspec";
+
+        private static readonly string[] Pdbs =
+        {
+            "src/FakeItEasy/bin/Release/net40/FakeItEasy.pdb",

```

Comment:
any idea why we don't do net45? I think this is pre-existing.

Reply:
No idea. We certainly should.

Reply:
Created #1415.